tests/Domain/RegularGivingNotifierTest.php

```diff
@@ -2,39 +2,145 @@
 use MatchBot\Domain\Salesforce18Id;
 use MatchBot\Domain\StripeCustomerId;
 use MatchBot\Tests\TestCase;
+use Prophecy\Prophecy\ObjectProphecy;
+use Psr\Clock\ClockInterface;
 use Ramsey\Uuid\Uuid;
+use Symfony\Component\Clock\Clock;
 use Symfony\Component\Clock\MockClock;
 
 class RegularGivingNotifierTest extends TestCase
 {
+    /** @var ObjectProphecy<Mailer> */
+    private ObjectProphecy $mailerProphecy;
+
+    /** @var PersonId  */
+    private PersonId $personId;
+
     public function testItNotifiesOfNewDonationMandate(): void
     {
-        $mailerProphecy = $this->prophesize(Mailer::class);
+        $donor = $this->givenADonor();
+        list($campaign, $mandate, $firstDonation, $clock) = $this->andGivenAnActivatedMandate($this->personId, $donor);
 
-        $clock = new MockClock('2024-12-01');
-        $sut = new RegularGivingNotifier($mailerProphecy->reveal(), $clock);
+        $this->thenThisRequestShouldBeSentToMatchbot([
+            "templateKey" => "donor-mandate-confirmation",
+            "recipientEmailAddress" => "donor@example.com",
+            "params" => [
+                "charityName" => "Charity Name",
+                "campaignName" => "someCampaign",
+                "charityNumber" => "Reg-no",
+                "campaignThankYouMessage" => 'Thank you for setting up your regular donation to us!',
+                "signupDate" => "01/12/2024 00:00",
+                "schedule" => "Monthly on day #12",
+                "nextPaymentDate" => "12/12/2024",
+                "amount" => "£64.00",
+//                "giftAidValue" => "", // see @todo in SUT
+//                "totalIncGiftAd" => "", // see @todo in SUT
+                "totalCharged" => "£64.00",
+                "firstDonation" => [
+                    // mostly same keys as used on the donorDonationSuccess email currently sent via Salesforce.
+                    'currencyCode' => 'GBP',
+                    'donationAmount' => 64,
+                    'donationDatetime' => '2025-01-01T09:00:00+00:00',
+                    'charityName' => 'Charity Name',
+                    'matchedAmount' => 64,
+                    'transactionId' => '[PSP Transaction ID]',
+                    'statementReference' => 'Big Give Charity Name'
+                    // @todo -- fill in properties below in implementation
+//                        'giftAidAmountClaimed' => 1_000, // see @todo in SUT
+//                        'totalCharityValueAmount' => 50_000, // see @todo in SUT
 
-        $campaign = TestCase::someCampaign(thankYouMessage: 'Thank you for setting up your regular donation to us!');
-        $personId = PersonId::of(Uuid::uuid4()->toString());
+                ]
+            ]
+        ]);
+
+        $this->whenWeNotifyThemThatTheMandateWasCreated($clock, $mandate, $donor, $campaign, $firstDonation);
+    }
+
+    public function markDonationCollected(Donation $firstDonation, \DateTimeImmutable $collectionDate): void
```

Comment:
I think a few supporting fns could be `private` to help see at a glance which are tests vs. test support?